services/src/main/java/org/jboss/windup/web/services/rest/UnhandledExceptionHandler.java

```diff
@@ -41,7 +41,7 @@ public Response toResponse(RuntimeException exception)
             errorInfo = new ErrorInfo(Response.Status.INTERNAL_SERVER_ERROR.getReasonPhrase());
 
             if (null != findFirstOccurenceInExceptionChain(org.hibernate.exception.ConstraintViolationException.class, exception))
-                errorInfo.setMessage("Database constraints were not met.");
+                errorInfo.setMessage("Something wrong happened on the server. The details are in the log.");
             else if (null != findFirstOccurenceInExceptionChain("org.h2.jdbc.JdbcSQLException", exception))
                 errorInfo.setMessage("Database error occurred.");
```

Comment:
Well my reasoning is that if the database error starts happening consistently, an advanced user may try to delete the database and start with a new one.